lib/node.js

```diff
@@ -458,6 +478,7 @@ class Node {
     }
 
     this.handle.release();
+    this._clock.handle.release();
```

Comment:
The clock's handle should preferably be a child of the node's handle to avoid needed to add this here. There's two approaches we could take.

1. Allow an optional parent handle in the Clock's constructor, then update rclnodejs.createClock to take in this handle.
2. Expose set_parent on RclHandle as a NAN_METHOD to allow updating a handle's parent from JavaScript.

2) is closer to rclpy's implementation, but I'm leaning more towards 1) as it keeps the handle management purely in c++.

Reply:
If you want to call `rcl_clock_fini()` immediately after this line, we can accept option 1. 

Unlike other entities, e.g. pub/sub/client/service, they must stop working and release the resources before the `node` is destroyed, so their parent has to be a concrete node.  The Clock doesn't depend on a `node`, maybe we can let the GC take care of the timing of doing this, so just `this._clock = null`, how do you think?

Reply:
I like the idea of setting `this._clock` to `null`. That's much simpler.